src/util/plotting.jl

```diff
@@ -0,0 +1,64 @@
+using RecipesBase
+
+@recipe f(gp::AbstractGP, x::Array) = gp(x)
+@recipe f(gp::AbstractGP, x::AbstractRange) = gp(x)
+@recipe f(gp::AbstractGP, xmin::Number, xmax::Number) = (gp, range(xmin, xmax, length=1000))
+@recipe function f(gp::FiniteGP; samples=0, sample_seriestype=:line)
```

Comment:
I guess the keyword arguments are not needed anymore:
```suggestion
@recipe function f(gp::FiniteGP)
```